Add What Are Connectors? explanation article (#2060)
dnsimple/dnsimple-support: content/articles/what-are-connectors.md

---
title: What Are Connectors?
excerpt: What a DNSimple connector does, what authorizing one grants, and how connectors differ from integrations and one-click services.
meta: A DNSimple connector links one domain to a Heroku app or Netlify site and configures the DNS records it needs. Learn what it authorizes and how it is monitored.
categories:
- Connectors
- Integrations
---

# What Are Connectors?

### Table of Contents {#toc}

* TOC
{:toc}

---

> A connector links one domain in your DNSimple account to one account on a third-party platform, creates the DNS records that platform needs, and then watches the connection to make sure it keeps working. DNSimple offers two connectors: Heroku and Netlify.

Reviewer: ```
creates the DNS records that platform needs
```

I think there might be a missing `the` before `platform`

> A connector links one domain in your DNSimple account to one account on a third-party platform, creates the DNS records that platform needs, and then watches the connection to make sure it keeps working. DNSimple offers two connectors: Heroku and Netlify.

Author: ```suggestion
A connector links one domain in your DNSimple account to one account on a third-party platform, creates the DNS records that the platform needs, and then watches the connection to make sure it keeps working. DNSimple offers two connectors: Heroku and Netlify.
```

Both authorizations are scoped to the connection you are creating. Revoking either one stops the connector from maintaining that connection.

## What a connection is {#connections}

Each connector you set up creates a connection, which is tied to your account and to one domain. A connection stores:

- A title identifying it
- Metadata about the connected app or site, which is what the connection card shows
- A history of messages about the connection
- A history of its status over time

You can see your connections on the <label>Connections</label> tab of a domain. The tab appears only for users who can manage the account.

## Connections are monitored {#monitoring}

DNSimple checks connections on an ongoing basis rather than only at setup. If a connection breaks, because the DNS records were removed or the configuration changed on the platform side, we send an email to your account notification email with instructions for fixing it and a link to reconnect.

This monitoring is what most distinguishes a connector from following a setup guide by hand.

## What happens when you delete a connection {#deletion}

> Disconnecting a connector removes the connection along with its stored metadata, messages, and status history.

Reviewer: When disconnecting a connector, the user can also choose to have  any of the associated DNS records and/or the hostname from the Heroku/Netlify app automatically deleted.

> Disconnecting a connector removes the connection along with its stored metadata, messages, and status history.

Author: ```suggestion
Disconnecting a connector removes the connection, and there is an option to have any of the associated DNS records and/or the hostname automatically deleted.
```

> [!NOTE]
> Removing a connection stops DNSimple from monitoring it. Review the DNS records for the domain afterwards to confirm they are what you want, and remove any that are no longer needed.

## Connectors, integrations, and services {#comparison}

These three overlap, and choosing between them is mostly a question of what you are connecting to.

| | What it is | Who maintains the DNS |
|---|---|---|
| **Connector** | An authorized link between one domain and one Heroku app or Netlify site | DNSimple writes the records and monitors the connection |

> | **Integration** | A provider-specific setup guide, or a supported external DNS provider configuration | Mostly you, following documented steps |

Reviewer: I wonder if it might be better to drop this row, since this is about the customer manually integrating things on their end, and not something offered by DNSimple

> | **Integration** | A provider-specific setup guide, or a supported external DNS provider configuration | Mostly you, following documented steps |

Author: ```suggestion
```
